src/commands/settings/handlers/tenderlyKeyHandler.ts

```diff
@@ -0,0 +1,25 @@
+import { getTenderlyKey, setTenderlyKey } from '../../../lib/keys.js';
+import { confirmPrompt, tenderlyKeyPrompt } from '../../../lib/prompts.js';
+import { viewHandler } from './viewHandler.js';
+
+export const tenderlyKeyHandler: (...args: unknown[]) => void | Promise<void> = async (
+  tenderlyKey: string,
+) => {
+  const existingTenderlyKey = await getTenderlyKey();
+
+  if (existingTenderlyKey) {
+    (await confirmPrompt('Overwrite existing Tenderly Key?'))
```

Comment:
"Seems like you already have a Tenderly key stored. Are you sure you want to override it with a new one?"